Hyper/Scene.cs

```diff
@@ -155,6 +163,14 @@ public void RegisterCallbacks()
         context.RegisterUpdateFrameCallback((e) =>
         {
             SimulationManager.Simulation.Timestep((float)e.Time, SimulationManager.ThreadDispatcher);
+            ResetRayCastingResult(ref RayCastingResults[0], Player);
+            SimulationManager.Simulation.RayCast(Player.RayOrigin, Player.RayDirection, Player.RayMaximumT, ref _hitHandler, 0);
         });
     }
+
+    private static void ResetRayCastingResult(ref RayHit hit, IRayCaster rayCaster)
```

Comment:
Moebe move the ray casting logic to a Physics handler?

Reply:
Certainly, I think that simulation manager would be a good place for that